config/crd/bases/sagemaker.services.k8s.aws_hyperparametertuningjobs.yaml

```diff
@@ -17,12 +17,12 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
-    - jsonPath: .status.failureReason
-      name: FailureReason
-      type: string
     - jsonPath: .status.hyperParameterTuningJobStatus
       name: HyperParameterTuningJobStatus
       type: string
+    - jsonPath: .status.failureReason
+      name: FailureReason
+      type: string
```

Comment:
@RedbackThomson didn't we have a fix for this non-deterministic sorting dilemma?

Reply:
I never worked on one, no. I looked into it based on your suggestion but it was non-trivial to me and I guess it was never prioritised.

Reply:
Create a github issue @RedbackThomson if there isnt one ?

Reply:
https://github.com/aws-controllers-k8s/community/issues/720 Opened 14 days ago

Reply:
@akartsky please git add only changes unrelated to this